src/app/views/demo/NestedActionDemo/nestedActions.ts

```diff
@@ -4,6 +4,7 @@ const setProfileAction = defineAction({
   id: 'set-profile',
   title: 'Set profile',
   subtitle: 'Select this and then choose one of the options',
+  shortcut: 'p o',
```

Comment:
suggestion: I'd prefer `p s` because it reads as `profile set`. I liked `profile open` less because `open` is very command palette specific whereas `set` is a little generic so if we provide alternative way to select profile this would still work.

Reply:
resolved @ https://github.com/itaditya/solid-command-palette/pull/15/commits/5d95e4f136ff94325e547076cb44f50adfce7fac